src/editor/plugins.js

```diff
@@ -104,6 +107,23 @@ export function getPlaceholderPlugin({ text }) {
     }
   }
 
+  function createPlaceholderNode(text) {
+    let span = document.createElement('span');
+    span.innerHTML = `
+<style>
```

Comment:
Also shouldn't make `::before` absolute, unless you're going to contain it in `span`, otherwise if `simpla-text` isn't pos relative the placeholder will jump out of context